### PR TITLE
Workflowforadapters

### PR DESCRIPTION
**Pull request by Revolyssup**
**Description**

This PR Added central reusable e2e test workflow for adapters

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit e7be30673b9c0ff2a0036e1dacfabea44720d63c at: https://619f46d1f14e9394deac7e2c--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit d80a92f06cb647a24f1e18a8b021bf488217113b at: https://619f49af2019c797b20b2206--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 3c9825059399d6f6e8b2f5169fafdc21a80a0e85 at: https://619fab9b96666100b2df989d--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 34780b4ae6cfe9d7037302c0866fa6ebd2624c73 at: https://619fce9ed5fb9a0078be1a6e--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 1ec04d5fb2b440d4be876b9a1774f1ca8fe04f9f at: https://61a08c4f7648a1033d0d4275--meshery-docs-preview.netlify.app

File: .github/workflows/testadapters.yaml [hunk 1]
```diff
@@ -0,0 +1,179 @@
+# This workflow will be called/referenced in each of the adapters and runs in the context of adapter
+name: Test for adapters using patternfile
```
Comment (leecalcote) [inline .github/workflows/testadapters.yaml:2]:
```suggestion
name: Adapter Integration: Core
```

**Review by leecalcote — APPROVED** [review]

**Review by leecalcote — APPROVED** [review]

**Pull request merged**